README.md

```diff
@@ -1,6 +1,16 @@
 ![MobiMart](https://socialify.git.ci/kavania2002/MobiMart/image?description=1&descriptionEditable=MobiMart%20is%20a%20mobile%20shopping%20website.%20Very%20useful%20for%20Django%20beginner.&font=Bitter&forks=1&issues=1&language=1&owner=1&pattern=Plus&pulls=1&stargazers=1&theme=Light)
 <br>
 
+# MobiMart
+<br>
+
+![Main Page ss1](https://user-images.githubusercontent.com/78464111/107855526-d4186000-6e48-11eb-9364-35e753427f8e.png)
```

Comment:
Add some more images of index page. The ones containing mobile phones.

Reply:
Ok
Thanks for suggestions